### PR TITLE
🐛teleport

### LINKED ISSUE #177: 🐛 Teleport gesture does not work sometimes

### PR DESCRIPTION
**Pull request by davey2206**
## Content

Fixed teleport bugs (see closes issues)

### Updated
`ProjectSettings/TagManager.asset` : Was updated to have a floor tag
`Assets/Scripts/Teleport/HandleHandTeleport.cs`: Was updated to disable teleport after x seconds
`Assets/Scenes/MainLevel.unity`: Updated Teleport areas
`Assets/Samples/XR Hands/1.4.1/Gestures/Examples/Hand Shapes/Point Hand Shape.asset`: Was updated to allow for easier teleporting
`Assets/Prefabs/PlayerRig/PlayerRig.prefab`: Was updated

## Testing

The feature / Bugfix can be testing by following these steps:

1. Open the scene
2. Play around with teleport

Closes #177, #181, #179, #180, #167, #168, #176

### CODE REVIEW

File: Assets/Scripts/Teleport/HandleHandTeleport.cs [hunk 1]
```diff
@@ -82,5 +88,22 @@ private async void EnableMovement(CancellationToken token)
...
         }
+
+        private async void DisableTimer(CancellationToken token)
+        {
+            try
+            {
+                await UniTask.WaitForSeconds(_timeBeforeDisable, cancellationToken: token);
+                DeactivateTeleport();
+
+                _cancellationTokenSourceTeleport.Cancel();
```
Comment (Timsel1) [inline Assets/Scripts/Teleport/HandleHandTeleport.cs]: 3 Lines of disposing and creating new token are in the wrong place, it should be here:
```c#
public void TriggerTeleport()
        {
            _alternateMovement.enabled = false;
            if (_isTeleportActive && _teleportInteractor.TryGetCurrent3DRaycastHit(out RaycastHit hit))
            {
                TeleportRequest request = new()
                {
                    destinationPosition = hit.point,
                };
                _teleportationProvider.QueueTeleportRequest(request);
            }
            _cancellationTokenSourceTeleport.Cancel();
            _cancellationTokenSourceTeleport.Dispose();
            _cancellationTokenSourceTeleport = new();

            EnableMovement(_cancellationTokenSource.Token);
        }
```

**Review by Timsel1 — CHANGES_REQUESTED** [review]
|Fix this|I have commented how to fix this|
|-|-|
|https://github.com/user-attachments/assets/898ad507-07de-4f94-b937-82de1f166776|https://github.com/user-attachments/assets/06b343bc-0afe-495c-bfee-446666e45444|

And still can't go through the door #168, can't event teleport through it anymore

File: Assets/Scripts/Teleport/HandleHandTeleport.cs (hunk 1 above)
Comment (davey2206) [reply Assets/Scripts/Teleport/HandleHandTeleport.cs]: done

**Review by LucVogelsangs — CHANGES_REQUESTED** [review]
Hole from start room to teleport room is walkable

**Review by LucVogelsangs — APPROVED** [review]

**Review by Timsel1 — APPROVED** [review]
hoop t

**Pull request merged**